egs/wsj/s5/steps/cleanup/create_segments_from_ctm_edits.py

```diff
@@ -0,0 +1,420 @@
+#!/bin/env python
+
+# Copyright 2016  Vimal Manohar
+# Apache 2.0
+
+from __future__ import print_function
+import sys, operator, argparse
+
+# Segment data using CTM and the edits from Levenshtein alignment of 'hypothesis' and 'reference':
```

Comment:
This comment does not do a very good job of explaining what this script is supposed to do.